app/views/api/v1/targets/index.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+json.targets @targets do |target|
```

Comment:
You can simplify this by doing 
```
json.targets do
  json.array! @targets, partial: 'info', as: :target
end
```

Reply:
It works but not getting the same output. Is missing the 'target' key for each object inside the array. Any clue?